docs/02-guides/display-prices-with-or-without-taxes.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: "Display prices with or without taxes"
+hide_title: true
+description:
+  This guide walks through the process of displaying prices with or without
+  taxes within your Front-Commerce application.
+---
+
+## Display prices with or without taxes
+
+<SinceVersion tag="3.9" />
+
+To display a price without taxes, you can add a new
```

Comment:
**nitpick:** I find this start a bit direct

**suggestion:** I'd suggest to add an introductory paragraph with the context of this feature. Example (to be improved):
```suggestion

## Why?

The default theme  (Theme Chocolatine) supports an app-wide configuration for how prices are displayed to users all around the application.

Technically, this is achieved through the `<PriceXXX>` component usage. Depending on your project's context, you might want to display all prices:
- without taxes (B2B contexts)
- with taxes (B2C contexts)
- or both

This guide explains how to achieve this.

## How?

To display a price without taxes, you can add a new
```